### ISSUE TITLE
hideEmpty is not working

### ISSUE THREAD

**Issue by ahmetkuslular** [atanasster/storybook-addon-deps#6]
```
import { addParameters } from "@storybook/react";
import { DocsPage } from 'storybook-addon-deps/blocks';

addParameters({
  docs: { page: DocsPage },
  dependencies: {
    withStoriesOnly: true,
    hideEmpty: true,
  }
});
```

>  "@storybook/react": "^5.3.13"
> "storybook-addon-deps": "^1.0.11",

Although I use it this way, I still see it in stories without DEPENDENCIES and DEPENDENTS.

**Comment by atanasster** [comment]
Hi @ahmetkuslular thanks very much for reporting this. I believe it's now fixed with "1.0.12" - please install the new version and let me know.

The issue was when no component is assigned to a docs page and the hideEmpty was not respected.

**Comment by ahmetkuslular** [comment]
thank u sir, fixed. 
> 1.0.12
